app/controllers/admin/accounts_controller.rb

```diff
@@ -14,6 +14,8 @@ def index
     def show
       authorize @account, :show?
 
+      flash.now[:notice] = I18n.t('accounts.instance_actor_flash') if @account.instance_actor?
```

Comment:
Hm, doesn't look like you removed it.

Reply:
Yeah I wrote “Kept the flash message in case someone stumbles upon it.”

Reply:
I think you misunderstand me, you set this flash and then you also render the string manually in the view like I suggested. So it will appear double

Reply:
oops! fixed!